pkgs/tools/audio/openai-whisper-cpp/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, SDL2
+, makeWrapper
+, wget
+}:
+
+stdenv.mkDerivation rec {
+  pname = "whisper.cpp";
+  version = "unstable-2022-12-07";
+
+  src = fetchFromGitHub {
+    owner = "ggerganov";
+    repo = pname;
+    rev = "3996ecc156486fb93ff505c01090d13192e72aa2";
+    sha256 = "sha256-hPRwjyARaJIjW6YSC7PdPgyao96lSHd5JGEtBLLM+Rg=";
+  };
+
+  patches = [ ./download-models.patch ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildInputs = [ SDL2 ];
```

Comment:
One of these (probably `buildInputs`) is going to need `darwin.apple_sdk.frameworks.Accelerate`, and possibly other frameworks, too. I can't test until the end of the week, though.

Reply:
FYI you need to add `darwin` as the function argument for the expression in the file, and then add the `Accelerate` value as I described, above.

With your change, I get this message.

```console
> nix build -L 'github:dit7ya/nixpkgs/openai-whisper-cpp#openai-whisper-cpp'
error: Function called without required argument "Accelerate" at /nix/store/946xw29yrsryccz32d1k9m74h4315awr-source/pkgs/tools/audio/openai-whisper-cpp/default.nix:7
```

As I said: if nobody else comes to it, I can take a look, myself, at the weekend, but before then I can't make time to do anything other than a few test runs.